lib/hutch/error_handlers/airbrake.rb

```diff
@@ -6,7 +6,8 @@ module ErrorHandlers
     class Airbrake
       include Logging
 
-      def handle(message_id, payload, consumer, ex)
+      def handle(properties:, payload:, consumer:, ex:)
```

Comment:
correct, I looked at your travis definition and seems we only care about ruby >= 2.0

```
  - "2.3.0"
  - "2.2"
  - "2.1"
  - "2.0"
  - "jruby-9.0.0.0"
```